hushline/settings.py

```diff
@@ -704,6 +719,40 @@ def update_smtp_settings() -> Response | str:
         flash("👍 SMTP settings updated successfully")
         return redirect(url_for(".index"))
 
+    @bp.route("/update-brand-primary-color", methods=["POST"])
+    @admin_authentication_required
+    def update_brand_primary_color() -> Response | str:
+        if (host_org := HostOrganization.default()) is None:
```

Comment:
I think we can just use `.one()` here and let the top level error handler report the full error and return 500 to the user. That tidies up the endpoint logic some, tho this is more a nit than a hard requirement.